docs/getting-started-local.md

```diff
@@ -371,3 +371,33 @@ I would love love love you to send me cool examples you've made, and I'm looking
 
 [@HaydenLee37](https://twitter.com/haydenlee37)
 
+--------
+
+#Stopping NAF
+
+How to stop NAF ?
+
+As you have observed there is no stop command. If you are in a Linux based server, e.g. Ubuntu, try the following commands to Find and Kill (the process that you have initiated for NAF).
+
+**1. Find**
+
+`sudo lsof -i:8080`
+
+With this command you find where node server listens (change 8080 to whatever port you use)
+
+Then you will get a big report that has also a line beginning with **node** 
+
+`COMMAND   PID        USER       FD   TYPE   DEVICE SIZE/OFF NODE NAME 
+
+node     3766        JohnSmith   20u  IPv4    27606      0t0  TCP *:8080 (LISTEN)`
+
+
+You want the second field namely ProcessID which happens for me to be 3766 but for you will be another number
+
+
+**2. Kill**
+
+**sudo kill -9 3766**
+
+
+
```

Comment:
On Linux, a process that runs in a terminal in foreground can be stopped with Ctrl+C.
Is this not working for you?

Reply:
The process might be initialized with a cron job or some other way. We usually put an automatic way in case for some reason the server restarts. So , in order to kill it you must search it.

Reply:
Ok. Generally pm2 is used to deploy a nodejs app, to start it as a service. Last time I deployed easyrtc-server.js on a test server I read https://www.digitalocean.com/community/tutorials/how-to-set-up-a-node-js-application-for-production-on-ubuntu-16-04 and used this:
```
$ sudo npm install -g pm2
$ pm2 start easyrtc-server.js
                          Runtime Edition

        PM2 is a Production Process Manager for Node.js applications
                     with a built-in Load Balancer.

                Start and Daemonize any application:
                $ pm2 start app.js

                Load Balance 4 instances of api.js:
                $ pm2 start api.js -i 4

                Monitor in production:
                $ pm2 monitor

                Make pm2 auto-boot at server restart:
                $ pm2 startup

                To go further checkout:
                http://pm2.io/


                        -------------

[PM2] Spawning PM2 daemon with pm2_home=/home/vincentfretin/.pm2
[PM2] PM2 Successfully daemonized
[PM2] Starting /home/vincentfretin/vr/easyrtc-server.js in fork_mode (1 instance)
[PM2] Done.
┌────────────────┬────┬──────┬────────┬───┬─────┬───────────┐
│ Name           │ id │ mode │ status │ ↺ │ cpu │ memory    │
├────────────────┼────┼──────┼────────┼───┼─────┼───────────┤
│ easyrtc-server │ 0  │ fork │ online │ 0 │ 0%  │ 29.6 MB   │
└────────────────┴────┴──────┴────────┴───┴─────┴───────────┘
 Use `pm2 show <id|name>` to get more details about an app


vincentfretin@server1:~/vr$ pm2 startup
[PM2] Init System found: systemd
[PM2] To setup the Startup Script, copy/paste the following command:
sudo env PATH=$PATH:/usr/bin /usr/lib/node_modules/pm2/bin/pm2 startup systemd -u vincentfretin --hp /home/vincentfretin

[PM2] Freeze a process list on reboot via:
$ pm2 save

[PM2] Remove init script via:
$ pm2 unstartup systemd

Show info about the app:

    pm2 show easyrtc-server

Show logs:

    tail -f /home/vincentfretin/.pm2/logs/easyrtc-server-out.log
```

Feel free to modify your doc to include that.